draft-ietf-ppm-dap.md

```diff
@@ -1461,6 +1461,19 @@ The Helper responds to the Leader with HTTP status code 201 Created and a body
 consisting of the `AggregationJobResp`, with media type
 "application/dap-aggregation-job-resp".
 
+Aggregation job initialization MUST be idempotent. That is, multiple requests to
+`PUT /tasks/{tasks}/aggregation_jobs/{aggregation-job-id}` for the same
+`aggregation-job-id` and the same request body MUST succeed. However, changing
+an aggregation job's parameters is illegal, so further requests to
+`PUT /tasks/{tasks}/aggregation_jobs/{aggregation-job-id}` for the same
+`aggregation-job-id` but with a different request body MUST fail with an HTTP
+client error status code.
```

Comment:
This seems to imply that if the Helper gets an agg request and marks the reports as processed, then in the second agg request it has to "forget" this previous state change and allow the reports to be processed. Is this intended?

Today, Daphne won't even let you retry the second request:
```rust
if !self
    .put_helper_state_if_not_exists(task_id, &agg_job_id, &state)
    .await?
{
    // TODO spec: Consider an explicit abort for this case.
    return Err(DapAbort::BadRequest(
        "unexpected message for aggregation job (already exists)".into(),
    ));
}
```

Another question: what would be the recommended mechanism for implementing this MUST? For multi-round VDAFs it would make sense to at least require the Helper to wait as long as possible before recording the reports that were processed (i.e., until the last request). But what if the aggregation job just takes one request/response cycle, e.g., if the VDAF is 1-round? 

Finally, how do we check that the message hasn't changed? We previously discussed keeping track of the hash of the request body. If we're going to require this then I think we need to spell it out.